js/popup.js

```diff
@@ -0,0 +1,172 @@
+    modalDescription: 'Calling all wordsmiths and storytellers! Our Blogging Platform is your creative playground. Share your thoughts, ideas, and adventures with the world. From captivating articles to thought-provoking discussions, our platform empowers you to express yourself like never before. Grab your virtual pen, unleash your imagination, and let your words fly!',
+  },
+  {
+    id: 5,
+    title: 'Fitness Tracker App',
+    subtitle: 'Track your fitness journey',
+    description: 'An application for tracking fitness activities and progress',
+    technologies: ['Vue.js', 'Firebase', 'CSS'],
+    imagePath: 'images/project6.jpg',
+    modalImagePath: './img/Snapshoot Portfolio.svg',
+    liveLink: 'https://rebelzob.github.io/Portfolio/',
+    sourceCodeLink: 'https://github.com/Rebelzob/Portfolio',
+    modalDescription: "Ready to crush your fitness goals? Our Fitness Tracker App is your ultimate companion on the journey to a healthier you. Track your workouts, set new personal records, and celebrate your progress. With interactive challenges, inspiring badges, and a supportive community, you'll stay motivated like never before. Get ready to unleash your inner fitness beast!",
+  },
+  {
+    id: 1,
+    title: 'E-commerce Website',
+    subtitle: 'Shop your favorite products online',
+    description: 'An online store for selling various products',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    imagePath: 'images/project2.jpg',
+    modalImagePath: './img/Snapshoot Portfolio.svg',
+    liveLink: 'https://rebelzob.github.io/Portfolio/',
+    sourceCodeLink: 'https://github.com/Rebelzob/Portfolio',
+    modalDescription: 'Get ready for an amazing online shopping experience! Our E-commerce Website is packed with the coolest products you can imagine. Explore our virtual aisles, browse through the trendiest items, and add them to your cart with just a click. Secure checkout, lightning-fast shipping, and jaw-dropping deals await you!',
+  },
+];
+
+function generateProjectCards() {
+  const worksSection = document.getElementById('works');
+
+  projects.forEach((project) => {
+    const card = document.createElement('article');
+    card.classList.add('single-card', 'flex-row', 'desktop-version', 'hover-background', 'hover-img');
+    let technologiesList = '';
+    project.technologies.forEach((tech) => {
+      technologiesList += `<li>${tech}</li>`;
+    });
+    card.innerHTML = `
+        <div>
+          <h4 class="card-title">${project.title}</h4>
+        </div> 
+        <div class="infography-card">
+          <p>${project.description}</p>
+        </div>
+        <div class="list-technologies-card">
+          <ul class="list-technologies-card-ul flex-row">
+            ${technologiesList}
+          </ul>
+        </div>
+        <button open-modal type="button" class="button-project-card flex-row">See project</button>
+      `;
+    worksSection.appendChild(card);
+  });
```

Comment:
Your design (Good job implementing the hover state for the project cards. 😊👍)
![image](https://github.com/Rebelzob/Portfolio/assets/78251927/021b9c37-abe8-40c3-a575-78678a4ce6f8)

Figma
![image](https://github.com/Rebelzob/Portfolio/assets/78251927/c46b9ba3-3f7f-473c-b518-59ad4941e7fe)

However, Looking at the design on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1), it is when the user hovers over each card that the button `See Project` appears. please let's correct that. 🤗🛠